conf/modules.config

```diff
@@ -40,8 +40,8 @@ process {
             enabled: params.save_db,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
-        ext.args = [
-            params.taxa_classification_mmseqs_db_savetmp ? "" : "--remove-tmp-files" ,
+        ext.args   = [
```

Comment:
Is that to align with `ext.prefix` and `publishDir`? Intense Harshil alignment! :tm: :smiley:

Reply:
These are from the new formatter that does partial harshil alignment